src/Akeneo/Category/back/Infrastructure/Symfony/Resources/config/jobs.yml

```
@@ -16,6 +16,20 @@ services:
         tags:
             - { name: akeneo_batch.job, connector: 'Cleaning categories enriched values', type: 'clean_categories_enriched_values' }
 
+    pim_catalog.job.clean_categories_template_enriched_values:
```

Naming suggestion for the job name, since we only clean one template & its enriched values
```suggestion
    pim_catalog.job.clean_category_template_and_enriched_values:
```